cdk-infra/lib/constructs/api/webhook-api-construct.ts

```diff
@@ -130,6 +137,15 @@ export class AutoBuilderApiConstruct extends Construct {
     jobsQueue.grantSendMessages(triggerLocalBuildLambda);
 
     // grant permission for lambdas to enqueue messages to the job updates queue
+    jobUpdatesQueue.grantSendMessages(slackTriggerLambda);
```

Comment:
Added this for the enhanced app changes. Basically, we need to send a message to the `jobUpdatesQueue` for the `inQueue` status. Previously, we would send a message to the `jobsQueue` which would handle this state in the `HandleJobs` function, but since we want long-running tasks, the ECS tasks will be listening to the `jobsQueue` instead.